pinc/pg.inc

```diff
@@ -3,26 +3,16 @@
 // Variables, functions, etc. (mostly URL-related) related to Project Gutenberg.
 
 $PG_home_url = "https://www.gutenberg.org/";
```

Comment:
It's more intuitive for URLs in variables to not have the training `/`. Consider:
```php
$url = "{$PG_home_url}ebooks";
$url = "$PG_home_url/ebooks";
```

I would suggest removing the trailing slash here and in `$PG_catalog_url`. It doesn't look like there would need to be any other code changes given how that variable is used -- except in `tools/project_manager/show_image_sources.php` which should be using `get_pg_catalog_url_for_etext()` anyway.

Reply:
Actually, there's also `get_pg_catalog_link_for_etext()`, which seems to work there.